dsn.go

```diff
@@ -665,3 +669,112 @@ func parseTimeout(value string) (time.Duration, error) {
 	}
 	return time.Duration(vv * int64(time.Second)), nil
 }
+
+// ConfigParam is used to bind the name of the Config field with the environment variable and set the requirement for it
+type ConfigParam struct {
+	Name          string
+	EnvString     string
+	FailOnMissing bool
+}
+
+// GetFromEnv is used to get the value of an environment variable from the system
```

Comment:
This function is needed in case it is required to set some configuration options not kept in the `Config`. It is used in `setMaxChunkDownloadWorkers` and `setCustomJSONDecoder` functions in `jsonresultset_test.go` and in the `GetConfigFromEnv` dsn function.

Reply:
I meant the comment :)

Reply:
Linter requires all the exported functions to have the comment explaining what the function does.

Reply:
Ok, I didn't know that!